modules/gapi/src/streaming/gstreamer/gstreamerinit.hpp

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2021 Intel Corporation.
+//
+// This software and the related documents are Intel copyrighted materials,
+// and your use of them is governed by the express license under which they
+// were provided to you ("License"). Unless the License provides otherwise,
+// you may not use, modify, copy, publish, distribute, disclose or transmit
+// this software or the related documents without Intel's prior written
+// permission.
+//
+// This software and the related documents are provided as is, with no
+// express or implied warranties, other than those that are expressly
+// stated in the License.
+//
+
+#ifndef OPENCV_GAPI_STREAMING_GSTREAMER_GSTREAMERINIT_HPP
+#define OPENCV_GAPI_STREAMING_GSTREAMER_GSTREAMERINIT_HPP
+
+namespace cv {
+namespace gapi {
+namespace wip {
+
+/*!
+ * \brief The GStreamerInit class
+ * Initializes gstreamer once in the whole process
+ * 
+ * 
+ * @note You need to build OpenCV with GStreamer support to use this class.
+ */
+class GStreamerInit
```

Comment:
GstreamerInit::init() is little bit confusing

maybe better is GStreamerEnv::init() or something  like that?